pkg/telemetryquerylanguage/contexts/metrics/README.md

```diff
@@ -0,0 +1,45 @@
+| metric.name                    | the name of the metric to which the data point being processed belongs                                        | string                                                                  |
+| metric.description             | the description of the metric to which the data point being processed belongs                                 | string                                                                  |
+| metric.unit                    | the unit of the metric to which the data point being processed belongs                                        | string                                                                  |
+| metric.type                    | the type of the metric to which the data point being processed belongs.  See enums below for integer mapping. | int64                                                                   |
+| metric.aggregation_temporality | the aggregation temporality of the metric to which the data point being processed belongs                     | int64                                                                   |
+| metric.is_monotonic            | the monotonicity of the metric to which the data point being processed belongs                                | bool                                                                    |
+| positive                       | the positive buckets of the data point being processed                                                        | pmetric.ExponentialHistogramDataPoint                                   |
+| positive.offset                | the offset of the positive buckets of the data point being processed                                          | int64                                                                   |
+| positive.bucket_counts         | the bucket_counts of the positive buckets of the data point being processed                                   | uint64                                                                  |
+| negative                       | the negative buckets of the data point being processed                                                        | pmetric.ExponentialHistogramDataPoint                                   |
+| negative.offset                | the offset of the negative buckets of the data point being processed                                          | int64                                                                   |
+| negative.bucket_counts         | the bucket_counts of the negative buckets of the data point being processed                                   | uint64                                                                  |
+
+## Enums
+
+The Metrics Context supports the enum names from the metrics proto.  In addition, it also supports an enum for metrics data type.
```

Comment:
Maybe along with using the proto defs, we could link these docs to the appropriate proto defs file and not duplicate the values here? We'd like to avoid getting bitten by this stuff.

Reply:
I'll link to the protos in this section as well.

Reply:
I linked to the protos in each README's enum section.  For metrics specifically I linked to the pdata numeric definitions for the data type enum, but we need to keep the table because the actual Enum Symbol is defined by the Metrics Context.